ui/src/components/Item/Preview.svelte

```diff
@@ -90,6 +90,7 @@
   {@const { title, blurb, description, image, color, link, createdAt } =
     getMeta(item)}
   <button
+    on:click
```

Comment:
is this intended?

Reply:
Yes - because the `<Preview>` element is not a real DOM element, passing an `on:click` to it won't actually do anything, but if the `<Preview>` element specifies `on:click` on one of it's children, the event defined by the parent is bound here. 

e.g
```svelte
// Parent.svelte
<Child on:click={doSomething} />
<Child on:click={doSomethingElse />

// Child.svelte
<button on:click />
```